move non translatable strings to according file and mark them as non-translatable

moving-bits opened the pull request:
- move non translatable strings to according file and mark them as non-translatable
- also format strings in strings_not_translatable to have the `translatable="false"` attribute in front consistently to make reading and editing of that file easier

moving-bits commented:
First try on moving all non-translatable string to the according source file and mark them as non-translatable. If you find more strings which should be marked as non-translatable, or strings wrongly sorted to this, please add as comment here, I will update the PR then.

Lineflyer commented on main/res/values/strings.xml:
```diff
@@ -1539,13 +1506,10 @@
-    <string name="facebook_title">Facebook</string>
     <string name="facebook_link"><a href="">c:geo page</a></string>
-    <string name="twitter_title">Twitter</string>
     <string name="market_title">Google Play</string>
```
Maybe also Google Play?

moving-bits replied:
I had not included Google Play as I was unsure whether it has the same name in all languages. - Just cross checked with current translation files: At least in `values-ko` variant it differs (see string `market_title`).

Lineflyer commented on main/res/values/strings.xml:
```diff
@@ -1651,13 +1614,6 @@
-    <string name="unit_ft">ft</string>
-    <string name="unit_yd">yd</string>
-    <string name="unit_mi">mi</string>
-
```
For these units I am unsure, whether they could be different in a different alphabet, like e.g. greek, cyrillic, etc?

moving-bits replied:
Ok, will revert this part - most languages use the original abbreviations, but some languages have translated the full names.

moving-bits commented:
> Mainly looks good to me. See my two comments inline

Thanks for cross-checking. I updated the PR and moved the unit strings back to the translatable strings, but left `market_title` unchanged, as we have at least one translation for it already.

Lineflyer commented:
OK, I will merge this if tests are green. Thanks for taking this point.

Lets see what Github does with branch ` moving-bits:issue_8206` after merge. Not sure if the new deletion setting also applies to remote repositories.

Lineflyer commented:
The auto deletion seems only to apply to the cgeo/cgeo repository. Remote branches are not autodeleted (which makes sense as it should be the responsibility of the repo owner).

moving-bits commented:
> The auto deletion seems only to apply to the cgeo/cgeo repository. Remote branches are not autodeleted (which makes sense as it should be the responsibility of the repo owner).

GitHub has been deleting remote branches earlier, starting at some point in (end of?) 2019, IIRC. But maybe only when I merged my own PR, not sure.

Lineflyer commented:
>  GitHub has been deleting remote branches earlier, starting at some point in (end of?) 2019, IIRC. But maybe only when I merged my own PR, not sure.

Yes, you should see an option in this PR to delete your local branch. I do not see it.

bekuno commented:
The unit strings are as
<string translatable="false" name="unit_yd">yd</string>

But this was planned to revert, isn't it?

moving-bits commented:
> The unit strings are as
> yd

That should be fine, as it's been like that before (and some language files indeed have translations for that).

But you triggered a different check for me, and I discovered a mistake I made while reverting the distance units back to `strings.xml`: I had not removed the `translatable="false"` flag. => Updated my PR now

bekuno commented:
@moving-bits 
With `back ticks` is my comment better understandable, see now https://github.com/cgeo/cgeo/pull/8211#issuecomment-613405339

bekuno commented:
> GitHub has been deleting remote branches earlier, starting at some point in (end of?) 2019, IIRC. But maybe only when I merged my own PR, not sure.

Maybe you had activated such autamatic for your repo? 
I have to delete all my merged PR manually.

moving-bits commented:
> With `back ticks` is my comment better understandable, see now [#8211 (comment)](https://github.com/cgeo/cgeo/pull/8211#issuecomment-613405339)

Thanks. Ok, so we've stumbled upon the same. This is fixed with #8212.

moving-bits commented:
> > GitHub has been deleting remote branches earlier, starting at some point in (end of?) 2019, IIRC. But maybe only when I merged my own PR, not sure.
> 
> Maybe you had activated such autamatic for your repo?

Not that I know of. When I started end of 2018, I had to delete the no longer needed branches locally (clone of my fork) and in my fork, both manually, same as you. (Therefore I created a little script to do this for me.)

At some point in time in 2019 GitHub started deleting the no longer needed branches in my fork automatically, so I only had them to manually delete locally since then.

moving-bits commented:
I just tested my theory of "this happens when I merge my own PR" with #8212.

=> Yes, this seems to be the cause.
After confirming the merge, GitHub automatically deleted my fork's branch:

![image](https://user-images.githubusercontent.com/3754370/79228711-350a8c80-7e62-11ea-872a-9ee3679f44a3.png)

bekuno commented:
Maybe you have forgotten your change at that point in your repo setting:
![grafik](https://user-images.githubusercontent.com/32555742/79229512-6fc0f480-7e63-11ea-8494-4fc452678f9f.png)
I is a nice feature :-)
